packages/tldraw/package.json

```
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "@stitches/react": "^1.2.8",
-    "@tldraw/core": "^1.14.0",
+    "@tldraw/core": "npm:@gamma-app/tldraw-core@latest",
```

Like we talked about IRL

```suggestion
    "@tldraw/core": "npm:@gamma-app/tldraw-core@1.15.0",
```